src/PowerState.cpp

```diff
@@ -15,24 +15,24 @@ PowerState::PowerState(){
   pinMode(SD_CARD_1_SS, OUTPUT);
   pinMode(SD_CARD_2_SS, OUTPUT);
   //misc
-  pinMode(FLUSH_NVS_SWITCH, INPUT);
+  pinMode(FLUSH_NVS_SWITCH, INPUT_PULLDOWN);
   pinMode(FLUSH_NVS_LED, OUTPUT);
-  pinMode(WIFI_STATION_SWITCH, INPUT);
+  pinMode(WIFI_STATION_SWITCH, INPUT_PULLDOWN);
   //turn on/off anything for basic operation
   this->enter_basic_state();
 }
 
-PowerState::~PowerState(){
-  delete this;
-}
+//PowerState::~PowerState(){
```

Comment:
I really should delete this before someone mistakenly tries to use it.  
  
Our chip is NOT happy when you try to `delete(this);` and breaks with a very cryptic error.